package.json

```diff
@@ -1,4 +1,7 @@
 {
+  "scripts": {
+    "start": "node ./scripts/yamlToReadmeConverter.js"
```

Comment:
`start` is probably not the best name for the readme generation. Something like `generate-readmes` and `check-readmes`?